site/en/rules/deploying.md

```diff
@@ -158,6 +180,12 @@ in the form of a library target located at `//<LANG>/runfiles` (an abbreviation
 of `//<LANG>/runfiles:runfiles`). User targets that need to access their data
 dependencies will typically add this target to their `deps` attribute.
 
+### Module extensions *
+
+If you provide [module extensions](/docs/bzlmod#module-extensions) to allow
+users of your rules to declare domain-specific external dependencies, these
+extensions should all be exported from a a file named `<LANG>/extensions.bzl`.
```

Comment:
`extensions.bzl` seems quite generic. Maybe `<lang>/bzlmod.bzl` instead?

Reply:
I actually think that generic is what we should go for here: `bzlmod` is just the codename for a currently experimental feature that is planned to become the only way for Bazel projects to declare their dependencies, after `WORKSPACE` has been deprecated and removed. By that time, `<LANG>/bzlmod.bzl` would look pretty weird and `<LANG>/extensions.bzl` would seem natural as everybody will be accustomed to module extensions. I agree that `bzlmod.bzl` would probably be more self-explanatory during this transition period, but IMO we should optimize for the long term.